src/mgmt/tests/mocks.c

```diff
@@ -168,6 +168,106 @@ mock_ds_mgmt_pool_delete_acl_teardown(void)
 	daos_acl_free(ds_mgmt_pool_delete_acl_result);
 }
 
+int				ds_mgmt_list_pools_return;
+char				ds_mgmt_list_pools_group[DAOS_SYS_NAME_MAX];
+void				*ds_mgmt_list_pools_npools_ptr;
+uint64_t			ds_mgmt_list_pools_npools;
+void				*ds_mgmt_list_pools_poolsp_ptr;
+struct mgmt_list_pools_one	*ds_mgmt_list_pools_poolsp_out;
+void				*ds_mgmt_list_pools_len_ptr;
+size_t				ds_mgmt_list_pools_len_out;
```

Comment:
nit: indented one tab too far?

Reply:
Checking the whitespace in my IDE, it looks OK, tabs are correct width. We've seen this a few times, not sure what's causing Github to display it differently.